packages/syntax/src/renderingPlugin.tsx

```diff
@@ -132,61 +195,71 @@ export function treeToLines(
     }
   });
 
-  // Strip empty lines from the beginning of code blocks
-  while (lines[0].length === 0) {
-    lines.shift();
-  }
-
-  // Strip empty lines from the end of code blocks
-  while (lines[lines.length - 1].length === 0) {
-    lines.pop();
-  }
-
   return lines;
 }
 
-interface TableRowLineProps {
-  lineNumber: number;
-  line: Array<TreeItem>;
+interface TableContentProps {
+  lines: Array<Array<TreeItem>>;
 }
 
-function TableRowLine({ lineNumber, line }: TableRowLineProps) {
-  return (
-    <LineTableRow lineNumber={lineNumber}>
-      {line.map(processToken)}
-    </LineTableRow>
-  );
-}
+export function TableContent({ lines }: TableContentProps) {
+  const { highlightLines, showLineNumbers, darkMode } = useSyntaxContext();
+  const trimmedLines = lines;
 
-interface TableRowFragmentProps {
-  line: Array<TreeItem>;
-}
+  // Strip empty lines from the beginning of code blocks
+  while (trimmedLines[0].length === 0) {
+    lines.shift();
+  }
+
+  // Strip empty lines from the end of code blocks
+  while (trimmedLines[trimmedLines.length - 1].length === 0) {
+    trimmedLines.pop();
+  }
 
-function TableRowFragment({ line }: TableRowFragmentProps) {
   return (
     <>
-      <span>{line.map(processToken)}</span>
-
-      {/* We use a new line character here instead of <br /> so that text will not break outside of a <pre /> tag */}
-      {'\n'}
+      {trimmedLines.map((line, index) => {
+        const currentLineNumber = index + 1;
+        const highlightLine = highlightLines.includes(currentLineNumber);
```

Comment:
do we polyfill `Array.prototype.includes`? IIRC it's not supported by IE

Reply:
I'm not sure, but we use it in more than a few components. If we need to address this, we probably need to do it everywhere.

Reply:
I'd be happy if we could formally drop support for IE! 🤞

Reply:
Me too...

In our babelrc on master, it looks like we're probably polyfilling this:
```js
const presets = [
    '@babel/preset-typescript',
    '@babel/preset-react',
    [
      '@babel/preset-env',
      {
        targets: {
          browsers: ['last 2 versions', 'safari >= 7', 'ie >= 10'],
        },
        modules: 'commonjs',
      },
    ],
  ];
```

Reply:
Correction: From looking at the built bundle itself, I can't find any indication that it's polyfilled.

Reply:
Also, updating the browsers list doesn't seem to result in any change to the bundle. it's pretty unclear what babel uses it for. maybe it only considers whether a browser supports specific syntaxes and not APIs.